packages/@tests/smoke/web-cra-ts/public/index.html

```diff
@@ -7,7 +7,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Web site created using create-react-app" />
-    <script src='http://localhost:8766/js-client.min.js' async></script>
+    <!-- Ideally we want to use 'async' here. Currently, it's not supported. -->
```

Comment:
could you please also add info why it's not supported so future you are able to understand this. Maybe even add task to linear. If I am understanding correctly it will not block while wasms are downloading so probably it's not that bad

Reply:
It's not supported because of current synchronous implementation. Previous implementation wasn't ideal - it was built upon setInterval for fetching library which is not ideal.

Reply:
yeah I remember setInterval was pretty hacky but Pavel valued `async` so he did that hack

Reply:
In current implementation async is not really important because main script is pretty lightweight compared to Pavel's script which inlined 2 heavy wasm files.

Reply:
You forgot to remove this comment